src/styles/index.css

```diff
@@ -0,0 +1,5 @@
+body {
+  margin: 0;
+}
+
+/*# sourceMappingURL=index.css.map */
```

Comment:
The presence of this `index.css` file suggests it might be a leftover or unnecessary if SCSS is being used. Consider removing this file if the SCSS is compiled and linked correctly in the HTML.